package-lock.json

```diff
@@ -15507,7 +15507,7 @@
       }
     },
     "packages/engine.io": {
-      "version": "6.6.3",
+      "version": "6.6.4",
```

Comment:
related to https://github.com/socketio/socket.io/commit/72d61dab822a2f932b828bbfdc500a566c7cc87f